examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -103,6 +170,20 @@ void RegularGridDecomposition::updateLocalBox() {
   }
 }
 
+void RegularGridDecomposition::initializeNeighbourIds() {
+  for (int i = 0; i < 3; ++i) {
+    auto neighbourIndex = i * 2;
```

Comment:
```suggestion
    auto neighborIndex = i * 2;
```
Everything else is AE too so please change all of the occurrences for consistency :/